custom_components/samsungtv_tizen/media_player.py

```diff
@@ -637,4 +713,10 @@ def media_previous_track(self):
             source_key = self._app_list[ source ]
             await self.hass.async_add_job(self.send_command, source_key, "run_app")
         else:
+            set_new_source = False
```

Comment:
We could remove `set_new_source` completely and just add `return` after `_LOGGER.error("Unsupported source")`, it will simplify the code and improve readability.

Reply:
Agree, and done